.changeset/tiny-foxes-provide.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-adr-backend': minor
+'@backstage/plugin-adr': minor
```

Comment:
Hm this might be potentially breaking since it's technically a change in (internal) behavior but any external breakage would be unintended so I think patch is still appropriate
```suggestion
'@backstage/plugin-adr': patch
```